src/main/java/hudson/security/LDAPSecurityRealm.java

```diff
@@ -1109,6 +1159,8 @@ private static String addPrefix(String server) {
      * See http://msdn.microsoft.com/en-us/library/aa746475(VS.85).aspx for the syntax by example.
      * WANTED: The specification of the syntax.
      */
+    @SuppressFBWarnings(value = "MS_SHOULD_BE_FINAL", 
+            justification = "Filter may be changed in the runtime by scripts if required")
```

Comment:
@stephenc 
I doubt it can be changed without breaking the backward compatibility. IMO it does not worth it